pandas/tests/extension/test_numpy.py

```diff
@@ -219,6 +219,12 @@ def test_getitem_scalar(self, data):
         super().test_getitem_scalar(data)
 
 
+# TODO Index.__new__ checks for PandasArray (and converts it explicitly), so the
+# monkeypatching to make PandasArray into a regular ExtensionArray doesn't work
+# class TestIndex(base.BaseIndexTests):
```

Comment:
We are explicitly checking for PandasArray in `Index.__new__`:

https://github.com/pandas-dev/pandas/blob/5ee8cf1daed481bdfd28399468a1233736332db3/pandas/core/indexes/base.py#L417-L419

So the current monkeypatching doesn't work here. 
(now, I commented this as "TODO", but I am not sure we should actually put much effort in "fixing" this)

Reply:
agreed let's just put a comment in the module-level docstring about why we're not implementing this test class